src/main/java/forestry/plugins/PluginFluids.java

```diff
@@ -145,21 +145,6 @@ public void doInit() {
                     items.refractoryEmpty.getItemStack(),
                     PluginCore.items.refractoryWax.getItemStack(),
                     0.10f);
-        }
-
-        FluidStack ethanol = Fluids.ETHANOL.getFluid(1);
-        GeneratorFuel ethanolFuel = new GeneratorFuel(
-                ethanol,
-                (int) (32 * ForestryAPI.activeMode.getFloatSetting("fuel.ethanol.generator")),
-                4);
-        FuelManager.generatorFuel.put(ethanol.getFluid(), ethanolFuel);
-
-        FluidStack biomass = Fluids.BIOMASS.getFluid(1);
-        GeneratorFuel biomassFuel = new GeneratorFuel(
-                biomass,
-                (int) (8 * ForestryAPI.activeMode.getFloatSetting("fuel.biomass.generator")),
-                1);
-        FuelManager.generatorFuel.put(biomass.getFluid(), biomassFuel);
```

Comment:
What about instead of removing fuel registration completely, make it conditional?
I. e.:
```
// pseudo-code
if (isEthanolRegisteredOrAllowedInConfig) {
	// register ethanol as fuel
}
if (isBiomassRegisteredOrAllowedInConfig) {
	// register biomass as fuel
}
// etc.
```

Reply:
i'm new to modding, but it sounds good